src/robot/libraries/BuiltIn.py

```diff
@@ -636,7 +636,7 @@ def should_be_equal(self, first, second, msg=None, values=True,
         | Should Be Equal | ${x} | expected | ignore_case=True | formatter=repr |
 
         ``formatter`` is new in Robot Framework 3.1.2 and ``strip_spaces`` is new
```

Comment:
"and" should be replaced by a comma here, but we could also remove the reference to old RF 3.1.2 version and the whole problems goes away. I'll do that myself so no changes needed.